src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -28,27 +28,31 @@ USAGE="
 usage: install_dependencies_and_build.sh [options]
 
 Installs Maven, Java JDK and builds Tritonserver Java bindings
--h|--help                  Shows usage
--t|--triton-home           Expected Trition library location, default is: /opt/tritonserver
--b|--build-home            Expected build location, default is: /tmp/build
--v|--maven-version         Maven version, default is: "3.8.4"
--c|--core-tag              Tag for core repo, defaut is: "main"
--j|--jar-install-path      Path to install the bindings .jar
---javacpp-branch           Javacpp-presets git path, default https://github.com/bytedeco/javacpp-presets.git
---javacpp-tag              Javacpp-presets branch tag, default "master"
+-h|--help                         Shows usage
+-t|--triton-home                  Expected Trition library location, default is: /opt/tritonserver
+-b|--build-home                   Expected build location, default is: /tmp/build
+-v|--maven-version                Maven version, default is: "3.8.4"
+-c|--core-tag                     Tag for core repo, defaut is: "main"
+-j|--jar-install-path             Path to install the bindings .jar
+--javacpp-branch                  Javacpp-presets git path, default is https://github.com/bytedeco/javacpp-presets.git
+--javacpp-tag                     Javacpp-presets branch tag, default "master"
+--include-developer-tools-server  Include C++ bindings from developer_tools repository
```

Comment:
What do you think about switching to using the word `enable`? This seems consistent with the verbiage we use elsewhere, most notably the build.py script. You might also be able to shorten this to just `enable-developer-tools`.

Reply:
developer-tools is the repo. The repo is for additional wrappers around Triton. The server folder is for the CAPI wrapper. I changed the enable keyword though.